apps/storybook/src/Overview.stories.mdx

```diff
@@ -14,65 +10,32 @@ import { Meta } from '@storybook/addon-docs';
   </h1>
 </center>
 
-## What is iTwinUI-react?
+This site demonstrates stories for all react components from [iTwinUI](https://github.com/iTwin/iTwinUI) v3. For more detailed guidance, visit the [documentation site](https://itwinui.bentley.com).
 
-iTwinUI-react is a React component library for [iTwinUI](https://github.com/iTwin/iTwinUI).
-The goal of this project is to provide React components that make it easier to use the styles from [`@itwin/itwinui-css`](https://github.com/iTwin/iTwinUI/tree/main/packages/itwinui-css). Check out the [demo website](https://itwin.github.io/iTwinUI/react) to see the components in action.
+If you're using an older version of iTwinUI, we highly recommend upgrading (see [migration guide](https://github.com/iTwin/iTwinUI/wiki/iTwinUI-react-v3-migration-guide)). In the meantime, you might find these links useful:
 
-## Installation
+- [v1 storybook](https://itwin.github.io/iTwinUI-react)
+- [v2 storybook](https://itwin.github.io/iTwinUI/legacy/v2/react)
 
-```
-npm install @itwin/itwinui-react
-```
-
-```
-yarn add @itwin/itwinui-react
-```
```

Comment:
since the plan is to phase out the public-facing aspect, i've shortened the readme, instead encouraging the use of documentation site. plus, installation instructions are the least useful part